cluster/images/etcd/Dockerfile

```diff
@@ -16,4 +16,6 @@ FROM BASEIMAGE
 
 EXPOSE 2379 2380 4001 7001
 COPY etcd* etcdctl* /usr/local/bin/
+RUN chmod +x /usr/local/bin/etcd* /usr/local/bin/etcdctl*
```

Comment:
one more minor suggestion: isn't the executable bit preserved by `COPY`? not using `RUN` makes cross building simpler, I wonder if we can just ensure +x on the host before copying.

Reply:
Moved to using `install` in Makefile.

Note that `chmod` is used in `/clusters/images/etcd-empty-dir-cleanup/Dockerfile` which was not modified in this PR.

Reply:
SGTM :-)